odl/solvers/functional/default_functionals.py

```diff
@@ -1217,18 +1223,23 @@ class KullbackLeiblerCrossEntropy(Functional):
 
     Notes
     -----
-    The functional :math:`F` is given by
+    The functional :math:`F` with prior :math:`g>0` is given by:
 
     .. math::
-        \\sum_{i} \left( g_i - x_i + x_i \log \left( \\frac{x_i}{ pos(g_i) }
-        \\right) \\right) + I_{x \\geq 0}(x)
-
-    where :math:`g` is the prior, and :math:`I_{x \\geq 0}(x)` is the indicator
-    function on nonnegative elements.
-
-    `Wikipedia article on Kullback Leibler divergence
-    <https://en.wikipedia.org/wiki/Kullback%E2%80%93Leibler_divergence>`_.
-    For further information about the functional, see for example `this article
+        F(x)
+        =
+        \\begin{cases}
+            \\sum_{i} \left( g_i - x_i + x_i \log \left( \\frac{x_i}{g_i}
+            \\right) \\right)
+            & \\text{if } x_i > 0 \\forall i
+            \\\\
+            +\\infty & \\text{else}
+        \\end{cases}
```

Comment:
The definition is the same, up to the fact that we switch which of the two arguments is seen as prior (i.e., is fixed) and which is seen as variable (i.e., the one we optimize over).

If you see it as a functional of two arguments
```
F(x, y) = \\begin{cases}
             \\sum_{i} \left( y_i - x_i + x_i \log \left( \\frac{x_i}{y_i}
             \\right) \\right)
             & \\text{if } x_i > 0 \\forall i \text{and } y_i > 0 \\forall i
             \\\\
             +\\infty & \\text{else}
         \\end{cases}
```
the one of the functionals is given by fixing `x`, the other by fixing `y`.